src/ecredis_server.erl

```diff
@@ -295,10 +309,16 @@ create_ets_tables(ClusterName) ->
 -spec lookup_eredis_pid(ClusterName :: atom(), Node :: #node{}) ->
     {ok, Pid :: pid()} | {error, any()}.
 lookup_eredis_pid(ClusterName, Node) ->
+    lookup_eredis_pid(ClusterName, Node, []).
```

Comment:
do we want this function?

Reply:
its called by 7 functions total – I didn't want to get rid of the arity 2 version for that reason